README.md

```diff
@@ -1,2 +1,54 @@
 # sumo-project
+
 This repository is used to track anything related to the platform that powers SUMO
+
+## Engineering Board
+
+Short description of the workflow and of the columns in the board.
+
+### Labels
+
+Each label in this board signifies a specific project.
```

Comment:
"project" being kitsune, m-response, etc. or "project" being responsive redesign, elastic, aaq, IA, etc.?

IMO milestones work better for the latter (you get a sense of progress, things can only belong to one) and labels work better for things that might cut across issues (sizing, bug, good-first-bug, etc.)

Reply:
Project being kitsune, respond something new along the way. Milestones need also to go in this and of course describe the process. I wanted to start with a skeleton README that we can build upon